a10_octavia/controller/worker/controller_worker.py

```diff
@@ -903,9 +903,13 @@ def perform_write_memory(self, thunders):
         store = {a10constants.WRITE_MEM_SHARED_PART: True}
 
         for vthunder in thunders:
+            deleteCompute = False
```

Comment:
We use underscores for vars not camel case!

Reply:
done